source/Ship.cpp

```diff
@@ -251,6 +251,7 @@ void Ship::FinishLoading()
 		explosionWeapon = &model->BaseAttributes();
 		pluralModelName = model->pluralModelName;
 		noun = model->noun;
+		baseAttributes.CopyTags(model->baseAttributes);
```

Comment:
Is this line necessary? If the ship definition being parsed has no `attributes` child, and this isn't the base model, then the base model's entire `baseAttributes` property is copied (in line 264 of this PR).